ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -925,6 +1034,8 @@ objects:
         value: /etc/openshift-installer/gce.json
       - name: KUBECONFIG
         value: /tmp/artifacts/installer/auth/kubeconfig
+      - name: ENABLE_MIRROR
+        value: ${ENABLE_MIRROR}
```

Comment:
nit: we don't actually need this in `teardown`, do we?

Reply:
We don't, fixed.